product_template_tags/models/product_template.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
It's not needed anymore in Python 3. You can check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

Reply:
@Trivedi-Vacha-SerpentCS fixed